src/Compute/Compute.Autorest/test/Invoke-AzSpotPlacementRecommender.Tests.ps1

```diff
@@ -0,0 +1,52 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Invoke-AzSpotPlacementRecommender'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Invoke-AzSpotPlacementRecommender.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Invoke-AzSpotPlacementRecommender' {
+    BeforeAll { 
+        $resourceSku1 = @{sku = "Standard_D2_v3"}
+        $resourceSku2 = @{sku = "Standard_D2_v2"}
+        $resourceSku3 = @{sku = "Standard_D32_v2"}
+        $desiredSizes = $resourceSku1,$resourceSku2,$resourceSku3
+        $desiredLocations = 'eastus','eastus2','westus'
+    }
+
+    It 'PostExpanded' {
+        {
+            $response = Invoke-AzSpotPlacementRecommender -Location eastus -DesiredCount 1 -DesiredLocation $desiredLocations -DesiredSize $desiredSizes
+        }
+    }
+
+    It 'Post' {
+        {
+            $spotPlacementRecommenderInput = 
+            @{
+                desiredSizes = $desiredSizes;
+                desiredCount = 100;
+                desiredLocations = $desiredLocations;
+                availabilityZones = $true
+            }
+            Invoke-AzSpotPlacementRecommender -Location eastus -SpotPlacementRecommenderInput $spotPlacementRecommenderInput -verbose
+        }
+    }
+
+    It 'PostViaIdentityExpanded' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
+
+    It 'PostViaIdentity' -skip {
```

Comment:
If you don't plan to write test for this parameter set, please remove the whole section. But we highly recommend testing each parameter set.

Reply:
Removed PostViaIdentity test